### PR TITLE
PEP 8 -- Style Guide for Python Code - contd

### PR DESCRIPTION
**Pull request by clementval**
So here is the second part of the PEP 8 convention formatting. 

I also added an information in the README of the python folder to follow this.

### CODE REVIEW

**Review by pspoerri — APPROVED** [review]

File: python/serialbox/serialization.py [hunk 1]
```diff
@@ -39,32 +40,35 @@
...
-        'double8' : np.float64,
-        'float648' : np.float64
+        'int1': np.int8,
+        'int2': np.int16,
+        'int4': np.int32,
+        'int8': np.int64,
+        'float4': np.float32,
+        'float324': np.float32,
+        'double8': np.float64,
+        'float648': np.float64
```
Comment (andyspiros) [inline python/serialbox/serialization.py:59]: Do we want to align the `:` as you did somewhere in ppser?

**Review by andyspiros — APPROVED** [review]
Thanks for these changes. I used to import the serializer with
`form serialization import serializer as Serializer`
thinking "We will probably capitalize its name". Now I will profit from this import statements!

**Comment by andyspiros** [comment]
"pspoerri approved these changes during the APN Info".

File: python/serialbox/serialization.py (hunk 1 above)
Comment (clementval) [reply python/serialbox/serialization.py:59]: @andyspiros Yeah, I'll align the `np` as the PEP 8 convention doesn't allow spaces before the `:`

**Pull request merged**